src/lib/registration/ui/not_installed_products_dialog.rb

```diff
@@ -56,7 +58,7 @@ def initialize
       end
 
       def run
-        Yast::Popup.Feedback(RegistrationUI::CONTACTING_MESSAGE, _("Checking product conflicts")) do
+        Yast::Popup.Feedback(RegistrationUI::CONTACTING_MESSAGE, REGISTRATION_CHECK_MSG) do
```

Comment:
You need to use `_()` with `N_()` as it just marks the text as translatable but does not actually do the translation. See http://yastgithubio.readthedocs.io/en/latest/localization/#translating-constants for more details.